client/src/components/Tips.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+  CodeBracketIcon,
+  LightBulbIcon,
+  PencilIcon,
+} from "@heroicons/react/24/solid";
+
+type TipProps = {
+  icon: React.ReactNode;
+  text: string;
+};
+
+const TipItem: React.FC<TipProps> = ({ icon, text }) => {
+  return (
+    <div className="w-1/4 interactive-bg-surface-container-high rounded-lg p-2 border border-outline">
+      {icon}
+      <div className="text-start my-2">{text}</div>
+    </div>
+  );
+};
+
+const Tips: React.FC = () => {
+  const tips: TipProps[] = [
+    {
+      icon: (
+        <CodeBracketIcon className="h-6 w-6 p-1 rounded-full bg-primary-container" />
```

Comment:
Looks good to me. I have some ideas which I'll implement later on in this file. But this is good.